src/functionHelper.js

```diff
@@ -9,21 +10,76 @@ const uuid = require('uuid/v4');
 const handlerCache = {};
 const messageCallbacks = {};
 
+function runPythonHandler(funOptions, options) {
```

Comment:
a little nitpicky, but this function name isn't quite sitting well with me. Correct me if I'm wrong, but this does not actually _run_ the python handler, and instead _returns_ the handler? If so, then maybe we could change it to `buildPythonHandler` or `get...` or something like that?